user/views.py

```diff
@@ -28,22 +31,35 @@ def authentication(request):
         else:
             return render(request, 'user/login.html', {'form': form})
 
-    return render(request, 'user/login.html', {'form': form})
+    else:
+        return render(request, 'user/login.html', {'form': form})
 
 
 # Method is called after userdata is entered and checks database for verification
 def authentic(request, username, password):
+    """Validates the user login details.
+    Parameters
+    ----------
+    request : request
+        to validate user login details.
```

Comment:
Capitalize and use a proper sentence. Also: This should be a description of the parameter (and not what is is good for)